epregressions/tests/test_energyplus.py

```diff
@@ -117,6 +118,7 @@ def test_eplus_passed_hvac_template(self):
         self.assertTrue(return_val[2])
         self.assertFalse(return_val[3])
 
+    @unittest.skipIf(sys.platform.startswith('win'), "GH Actions is having trouble executing dummy.epmacro")
```

Comment:
These two unit tests were not running on CI, even though they run locally.  I will try to debug with an upterm session sometime, but for now it is fine.  They run well on the other platforms on CI and they run locally on Windows.